cpp/src/slice2swift/SwiftUtil.h

```diff
@@ -8,94 +8,78 @@
 
 using StringPairList = std::list<std::pair<std::string, std::string>>;
 
-namespace Slice
+namespace Slice::Swift
 {
-    const int TypeContextInParam = 1;
-    const int TypeContextProtocol = 2;
-
     std::string getSwiftModule(const ModulePtr&, std::string&);
     std::string getSwiftModule(const ModulePtr&);
 
-    class SwiftGenerator
-    {
-    public:
-        SwiftGenerator() = default;
-        SwiftGenerator(const SwiftGenerator&) = delete;
-        virtual ~SwiftGenerator() = default;
-
-        SwiftGenerator& operator=(const SwiftGenerator&) = delete;
```

Comment:
The diff is really struggling here. All I changed was removing these constructors, destructors, and operators.

For the other methods in this file, I removed `static` if it was present, and dedented them.
I didn't change anything else about them.